src/dashboard/osdeps/Ubuntu-14.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_comments": [
+    "Add any comments here",
+    "Include here keys, repos, and packages required by this component not already in archivematicaCommon osdeps file"
+  ],
+  "osdeps_repokeys": [
+  ],
+  "osdeps_repos": [
+  ],
+  "osdeps_packages": [
+  { "name": "gettext", "state": "latest"}
```

Comment:
There are several dependencies listed in the control file that are not in here - are they handled elsewhere?